### PR TITLE
CORTX-31633: Codacy code cleanup(#1824)

### PR DESCRIPTION
**Pull request by alfhad**
This patch fixes some of the codacy warnings.
Warning fixed: "W0702: No exception type(s) specified (bare-except)"

Signed-off-by: alfhad <fahadshah2411@gmail.com>

# Problem Statement
We see 5 occurrence of pattern, "bare-except"

# Design
Added the following comment to suppress the warning:
```# pylint: disable=bare-except```

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

**Review by nikitadanilov — CHANGES_REQUESTED** [review]
This patch masks the problem instead of fixing it. The right solution is to use the proper exception classes in 'except'.

**Comment by alfhad** [comment]
It is not possible to find which class of Exception will be raised for each of the the except type
Hence, we can suppress it by adding the comment or it can be simply ignored

File: scripts/addb-py/chronometry/addb2db.py
Comment (rkothiya) [inline scripts/addb-py/chronometry/addb2db.py:636]: you might want to catch exceptions like : 
```
except FileNotFoundError:
    print(f"Error could not find {file}") 
except IOError:
    print(f"Unable to read {file} check permissions")
finally:
    print(f"Error app reading file {file}")
```    
Before writing above please check about the syntax of IOError exception.

Comment (alfhad) [reply scripts/addb-py/chronometry/addb2db.py]: I have addressed the exceptions which can be known.

**Comment by alfhad** [comment]
Since it is not possible to anticipate all the Exceptions that will occur I have added a print statement to print the Exception occurred which should make debugging easier in case if an unanticipated exception occurs.

**Comment by chandradharraval** [comment]
Hi @rkothiya ,
Have we ignored above issue and can be closed?

**Comment by rkothiya** [comment]
@chandradharraval we need approval from the team that we agree to Alfhad's point and we can ignore this. Once we get that approval then we can ignore it in codacy.

**Comment by rkothiya** [comment]
Since all the changes that are done are in chronometry and all the warnings under chronometry were disabled because this code was shifted in another repository seagate-tools. 

So these changes are no longer needed.

**Pull request closed without merging**
